### ISSUE TITLE
getImage ignores "exportable: false" configs

### ISSUE THREAD

**Issue by CarlosHdez** [amcharts/amcharts4#1045]
I'm trying to export a chart using the API, my chart has a scrollbar that I want removed from the export so it has
```
{
   data: [],
   ...
   scrollbarX: {
      type: 'Scrollbar',
      exportable: false
   }
}
```
When I use the exporting menu the resulting image does ignore the scrollbar, but using `this.chart.exporting.getImage('png')` has it in it.
Is there a configuration I'm missing, or maybe an issue with JSON-based config?

**Comment by martynasma** [comment]
Nope, hiding of non-exportable items happens outside `getImage()`.

There are methods that do that, but they are private.

We will need to think whether maybe we need to move those inside `getImage()`.

Will keep you posted. Thanks for bringing it to our attention!

**Comment by CarlosHdez** [comment]
Cool, thanks

**Comment by martynasma** [comment]
Fixed in [4.2.2](https://github.com/amcharts/amcharts4/releases/tag/4.2.2).

## [4.2.2] - 2019-03-13

### Added
- `cursorHoverEnabled` added to `XYSeries`. Controls whether to trigger hover state on columns/bullets of the all series under hovered category/date.
- A new `plugins` property on all `Sprite`. [More info](https://www.amcharts.com/docs/v4/concepts/plugins/).
- New plugin: Regression. [More info](https://www.amcharts.com/docs/v4/tutorials/regression-trend-lines/).
- New `ExportMenu` setting: `closeOnClick` (default `true`). Will force export menu to close when export is initiated.

### Changed
- `DateFormat` will now parse even partial dates, e.g. format `"yyyy-MM-dd"` will parse `"2018"` correctly, and will assume Jan 1st.

### Fixed
- Export: `getImage()`/`getSVG()` will now correctly omit `exportable = false` elements.
- Export: Data export was not taking in additional columns after data updates.
- Some chart cloning issues fixed.
- Inccorrect behavior of Series' tooltip with `minBulletDistance` set fixed.
- Sometimes, when series had only one data item, chart used to zoom-in right after the init.
- In-line date formatting will now use `DateFormatter`'s parsing functions for string-based dates.
- `DateAxis` issues with certain timezones fixed.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
